specs/pages/failing/004.md

```diff
@@ -1,6 +1,6 @@
 # du
 
-> Secretly,
+> Secretly
```

Comment:
Should introduce a test that confirms we're issuing a warning when a line ends with `,` or `:` along with failing TLDR004.

Reply:
I don't think I have enough experience with the linter to add this functionality and catch the `console.warn`.

Reply:
Created #135 to track this.